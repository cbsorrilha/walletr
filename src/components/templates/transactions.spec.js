import React from 'react';
import { render } from '@testing-library/react';
import Transactions from './transactions';

describe('Test the transactions component', () => {
  it('should match the snapshots', () => {
    const component = render(<Transactions transactions={[]} />);
    expect(component).toMatchSnapshot();
  });
});
