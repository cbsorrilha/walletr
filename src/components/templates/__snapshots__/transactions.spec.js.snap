// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the transactions component should match the snapshots 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="sc-bdVaJa gWRmHU"
        color="#086a2d"
      >
        Seu extrato
      </h1>
      <div
        class="sc-dnqmqq lexibb"
      >
        <div
          class="sc-gzVnrw eJrXoB"
          direction="column"
        >
          <div
            class="sc-htoDjs fVLvim"
          >
            <div
              class="sc-gzVnrw kmpXob"
              direction="column"
            >
              <div
                class="sc-htoDjs fVLvim"
              >
                <h1
                  class="sc-bdVaJa bjHdda"
                  color="#086a2d"
                >
                  Nada aqui por enquanto.
                </h1>
              </div>
              <div
                class="sc-htoDjs eskNEf"
              >
                <p
                  class="sc-bZQynM jmyuGj"
                  color="#086a2d"
                >
                  Por que não experimenta comprar algumas bitcoins?
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <h1
      class="sc-bdVaJa gWRmHU"
      color="#086a2d"
    >
      Seu extrato
    </h1>
    <div
      class="sc-dnqmqq lexibb"
    >
      <div
        class="sc-gzVnrw eJrXoB"
        direction="column"
      >
        <div
          class="sc-htoDjs fVLvim"
        >
          <div
            class="sc-gzVnrw kmpXob"
            direction="column"
          >
            <div
              class="sc-htoDjs fVLvim"
            >
              <h1
                class="sc-bdVaJa bjHdda"
                color="#086a2d"
              >
                Nada aqui por enquanto.
              </h1>
            </div>
            <div
              class="sc-htoDjs eskNEf"
            >
              <p
                class="sc-bZQynM jmyuGj"
                color="#086a2d"
              >
                Por que não experimenta comprar algumas bitcoins?
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
